main/ertp/ERTP-Introduction.md

```diff
@@ -0,0 +1,117 @@
+# ETRP Introduction
+
+ERTP (*Electronic Rights Transfer Protocol*) is Agoric's token
+standard for transferring tokens and other digital assets in
+JavaScript. Using the ETRP API, you can easily create digital assets,
+all of which are transferred exactly the same way and with exactly the
```

Comment:
I think that this "transferred exactly in the same way..." part doesn't make sense unless you make it clear to the user that a wide variety of digital assets can be represented this way. If the user thinks that only a narrow subset of assets (like ERC20 tokens) can be represented, they will think the fact that the assets are treated in the same way is a given, and this is a weird point.